python/ray/data/datasource/parquet_datasource.py

```diff
@@ -419,31 +423,28 @@ def _fetch_metadata(
 
 
 def _sample_piece(
-    file_piece: _SerializedPiece,
-    row_group_id: int,
+    reader_args, columns, schema, file_piece: _SerializedPiece,
 ) -> float:
-    # Sample the `row_group_id`-th row group from file piece `serialized_piece`.
+    # Sample the first rows batch from file piece `serialized_piece`.
     # Return the encoding ratio calculated from the sampled rows.
     piece = _deserialize_pieces_with_retry([file_piece])[0]
 
-    # If required row group index is out of boundary, sample the last row group.
-    row_group_id = min(piece.num_row_groups - 1, row_group_id)
-    assert (
-        row_group_id >= 0 and row_group_id <= piece.num_row_groups - 1
-    ), f"Required row group id {row_group_id} is not in expected bound"
-
-    row_group = piece.subset(row_group_ids=[row_group_id])
-    metadata = row_group.metadata.row_group(0)
-    num_rows = min(PARQUET_ENCODING_RATIO_ESTIMATE_NUM_ROWS, metadata.num_rows)
```

Comment:
This constant is no longer used, right?

https://github.com/ray-project/ray/blob/699f2d6aa43d883272cc8aef2ef09fa4aa60b689/python/ray/data/datasource/parquet_datasource.py#L67

Reply:
Ah I see that you removed it in a recent commit.